components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/impl/OperationPoliciesApiServiceImpl.java

```diff
@@ -219,20 +210,20 @@ public Response deleteCommonOperationPolicyByPolicyId(String operationPolicyId,
      * @param messageContext message context
      * @return A list of operation policies available for the API
      */
-    @Override
-    public Response getAllCommonOperationPolicies(Integer limit, Integer offset, String query,
+    @Override public Response getAllCommonOperationPolicies(Integer limit, Integer offset, String query,
             MessageContext messageContext) throws APIManagementException {
 
         String apiManagementExceptionErrorMessage = "";
         OperationPolicyDataListDTO policyListDTO = null;
-        String name = null, version = null;
+        String name = null;
+        String version = null;
         try {
             String organization = RestApiUtil.getValidatedOrganization(messageContext);
             APIProvider apiProvider = RestApiCommonUtil.getLoggedInUserProvider();
 
             // If name & version are given, it returns the policy data
             if (query != null) {
-                Map<String, String> queryParamMap = new HashMap();
+                Map<String, String> queryParamMap = new HashMap<String, String>();
```

Comment:
Why changed this?
Explicit type argument String, String can be replaced with <>

Reply:
Does not exist in the latest one